cypress/integration/leaderboard_spec.js

```diff
@@ -0,0 +1,37 @@
+const url = Cypress.config().baseUrl;
+
+let score;
+
+describe('Leaderboard', () => {
+    describe('on logging in for the first time', () => {
+        beforeEach(() => {
+            cy.login('unicorn')
+            .then(() => cy.getPlayers());
```

Comment:
Can this be goToLeaderboard or something?

Reply:
It's just cause the request it's executing is getting the players (because that's called when the leaderboard component mounts), so I figured it'd be a more appropriate name but yeah sure

Reply:
Yeah either is fine I just couldn't tell what it did from the name